CMSgov/design-system
# [WNMGDS-1733] Add Gatsby component for rendering theme-specific content

## Summary

https://jira.cms.gov/browse/WNMGDS-1733

### Added

- Added a `<ThemeContent>` component that can be wrapped around MDX content to only render it when the current theme matches the conditions specified

### Changed

- Updated the instances of `< ComponentThemeOptions>` in our MDX content to match `<ThemeContent>` in its anticipation of knowing the current `theme` intrinsically when we implement the theme switcher. For now it's being hardcoded into the `ContentRenderer`

## How to test

- See the example in 5c8514d0b9ea8a9a9bc5663db46a54ce2a1fbe0f for content being wrapped in one of these. Try it out yourself. Pay attention to the doc comments for the component that explain the extra spacing needed for markdown rendering to function properly
